components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Stack/Encoders/Models/EncodeableDictionary.cs

```diff
@@ -6,12 +6,18 @@
 namespace Opc.Ua.Encoders {
     using Opc.Ua;
     using System.Collections.Generic;
+    using System.Linq;
 
     /// <summary>
     /// Encodeable dictionary carrying field names and values
     /// </summary>
     public class EncodeableDictionary : List<KeyDataValuePair>, IEncodeable {
 
+        /// <summary>
+        /// Identifier for the keys in the reversible format.
+        /// </summary>
+        private const string kKeysIdentifier = "_Keys";
```

Comment:
kKeysIdentifier? Is that the correct format?

Reply:
Good question. Actually yes (from .editorconfig):
![image](https://user-images.githubusercontent.com/2373875/115551325-2b083e80-a2ab-11eb-916f-c48087aa146c.png)